api/public_api_v1.yaml

```diff
@@ -1,8 +1,101 @@
 swagger: '2.0'
 info:
-  version: 0.1.0
+  version: 1.0.0
   title: Digital Citizenship API
-  description: Digital Citizenship API.
+  contact:
+    name: Digital Transformation Team and AgID
+    url: https://forum.italia.it/c/progetto-io
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
+    # Introduction
+
+    This is the documentation of the Digital Citizenship API.
+    This API enables Public Administration services to integrate with the
+    Digital Citizenship platform of the Italian Government.
+    The platform enables services to communicate with Italian Citizens via
+    the [IO app](https://io.italia.it/).
+
+    # How to get an API key
+
+    To get access to this API, you'll need to register on the
+    [Digital Citizenship Developer Portal](https://developer.cd.italia.it/).
+
+    Once registered, you will receive an API Key (`Ocp-Apim-Subscription-Key`)
```

Comment:
note that after registration, you have to click 'subscribe to the digital citizenship api' button